washing-up.md

```diff
@@ -0,0 +1 @@
+# Washing up
```

Comment:
I have nothing to add, everything looks good , thanks

Reply:
Thank you brother